### DISCUSSION TITLE
Google地圖怎麼用ＡＩ幫你判斷最佳路徑

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#738] [General]
不論我們去外地旅遊又或是在本地遊玩
都很常需要google map的幫助
因為他會在最有效率的情況下幫你分析怎麼走最好
將龐雜的五種交通方式綜合分析
我覺得這個技術實在太厲害了
所以查了資料分享給大家
https://technews.tw/2020/09/04/how-google-maps-navigation-works/

**Comment by HsuChen88** [comment]
原來Google Map已經從原本的路線規劃，缺點在於只有初始車流量評估，隨時間車多了最佳路線會改變；後來加入及時車流量，隨機應變；現在直接用根據平時的車流量預測未來的車流量來規劃路線，可以說是有點東西。
